### ISSUE TITLE
Library Licence

### ISSUE THREAD

**Issue by felipewer** [LabSEC/libcryptosec#7]
Add licence file to the project.

**Comment by lucasperin** [comment]
Yep, have to talk to some people about which license we are going to use!

**Comment by lucasperin** [comment]
OpenSSL is BSD-like. I have included a BSD 3 license from the github templates.
I think this is ok for now, though, it might have to change later.

**Comment by lucasperin** [comment]
Just noticed.. spec file had GPL license from ages ago..
Now what?
